src/main/java/com/redhat/devtools/intellij/commonuitest/UITestRunner.java

```diff
@@ -168,63 +174,64 @@ public int toInt() {
 
     private static void makeSureAllTermsAndConditionsAreAccepted() {
         String osName = System.getProperty("os.name").toLowerCase();
+        String userHome = "user.home";
 
         if (osName.contains("linux")) {
             step("Copy the 'prefs.xml' file to the appropriate location", () -> {
                 String prefsXmlSourceLocation = "prefs.xml";
-                String prefsXmlDir = System.getProperty("user.home") + "/.java/.userPrefs/jetbrains/_!(!!cg\"p!(}!}@\"j!(k!|w\"w!'8!b!\"p!':!e@==";
+                String prefsXmlDir = System.getProperty(userHome) + "/.java/.userPrefs/jetbrains/_!(!!cg\"p!(}!}@\"j!(k!|w\"w!'8!b!\"p!':!e@==";
```

Comment:
Wouldn't it be easier to just define a constant USER_HOME than to call over and over System.getProperty(userHome) ?

Reply:
Fixed.